### PR TITLE
Added softmax accelerator

### PR DESCRIPTION
**Pull request by belanoa**

### CODE REVIEW

File: Bender.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ dependencies:
   redundancy_cells:       { git: "https://github.com/pulp-platform/redundancy_cells.git", rev: c37bdb47339bf70e8323de8df14ea8bbeafb6583 } # branch: astral_rebase
   redmule:                { git: "https://github.com/pulp-platform/redmule.git", rev: 7abb256 } # branch: fc/hci-v2
   neureka:                { git: "https://github.com/pulp-platform/neureka.git", rev: 60c51c0 } # branch: hci-v2
-  ez_sfm:                 { git: "https://github.com/belanoa/ez_sfm.git"       , rev: 7630cbe } # branch: hci-v2
+  softex:                 { git: "https://github.com/belanoa/softex.git"       , rev: 9da2870 }
```
Comment (yvantor) [inline Bender.yml:38]: Is this the master branch? Could you please add as a comment the name of the branch it comes from?

**Review by yvantor — COMMENTED** [review]
There is no regression test added for the accelerator. I think this is required before merging (once we have it, we can also add it to the CI)

File: Bender.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@ dependencies:
   redundancy_cells:       { git: "https://github.com/pulp-platform/redundancy_cells.git", rev: c37bdb47339bf70e8323de8df14ea8bbeafb6583 } # branch: astral_rebase
   redmule:                { git: "https://github.com/pulp-platform/redmule.git", rev: 7abb256 } # branch: fc/hci-v2
   neureka:                { git: "https://github.com/pulp-platform/neureka.git", rev: 60c51c0 } # branch: hci-v2
+  softex:                 { git: "https://github.com/belanoa/softex.git"       , rev: 9da2870 } # branch: master
```
Comment (FrancescoConti) [inline Bender.yml:38]: Before merging it's better to wait for this to be public (and possibly in the pulp namespace), do you agree @yvantor ?

**Review by FrancescoConti — CHANGES_REQUESTED** [review]
The change I request is only the open sourcing of course ;)

**Comment by yvantor** [comment]
Closing as tackeld in #71.

**Pull request closed without merging**